packages/shaderlab/src/shaders/shadingPBR/BRDF.glsl

```diff
@@ -225,6 +329,12 @@ void initBRDFData(SurfaceData surfaceData, out BRDFData brdfData){
         brdfData.clearCoatRoughness = max(MIN_PERCEPTUAL_ROUGHNESS, min(surfaceData.clearCoatRoughness + getAARoughnessFactor(surfaceData.clearCoatNormal), 1.0));
         brdfData.clearCoatSpecularColor = vec3(0.04);
     #endif
+
+    #ifdef MATERIAL_ENABLE_IRIDESCENCE
+        float topIor = 1.0;
+        float NdotV = saturate(dot(surfaceData.normal, surfaceData.viewDir ) );
+        brdfData.iridescenceSpecularColor = EvalIridescence(topIor, NdotV, surfaceData.iridesceceIor, brdfData.specularColor, surfaceData.iridescenceThickness);   
```

Comment:
variables and methods start with lowercase:
so modify `EvalIridescence` to `evalIridescence`